tests/drivers/spi/spi_loopback/boards/apard32690_max32690_m4.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1a_miso_p1_28 {
+	power-source = <MAX32_VSEL_VDDIOH>;
+};
+
+&spi1a_mosi_p1_29 {
+	power-source = <MAX32_VSEL_VDDIOH>;
+};
+
+&spi1a_sck_p1_26 {
+	power-source = <MAX32_VSEL_VDDIOH>;
+};
+
+&spi1a_ss0_p1_23 {
+	power-source = <MAX32_VSEL_VDDIOH>;
+};
+
+arduino_spi: &spi1 {
+	pinctrl-0 = <&spi1a_miso_p1_28 &spi1a_mosi_p1_29 &spi1a_sck_p1_26
+		     &spi1a_ss0_p1_23>;
+	pinctrl-names = "default";
+	compatible = "adi,max32-spi";
```

Comment:
These lines are redundant to `boards/adi/apard32690/apard32690_max32690_m4.dts`, so you don't need to redefine them again here.